scripts/pymultic

```diff
@@ -142,6 +156,35 @@ def acquire_python(snekdir, version):
     return pyexe
 
 
+def build_python_container(snekdir, version):
+    realver = PYVERS[version]
+    snek = 'Python-{}'.format(realver)
+    builddir = os.path.join(snekdir, snek)
+    container_exe = get_container_exe()
+    py_container_tag = 'python:{}'.format(realver)
+
+    if subprocess.call([container_exe, 'image', 'inspect', py_container_tag],
+                       stdout=subprocess.DEVNULL,
+                       stderr=subprocess.DEVNULL) == 0:
+        return
+
+    fetch_python(snekdir, version)
+
+    print('Building Python {} container...'.format(realver))
+    logfile = os.path.join(snekdir, '{}.build.log'.format(snek))
+    install_target = 'fullinstall' if version == '3.0' else 'install'
```

Comment:
Interesting...  Why the difference here for 3.0?

Reply:
Yeah, this was an interesting find. I'm guessing that it's just a 2.x -> 3.x transitional oddity. Running `make install` from the 3.0 release installs the binary as `python3` and gives you a warning about not wanting to trample on the 2.x `python` binary. It goes on to say that if you want this installed as `python`, then you need to run `make fullinstall`. That target doesn't seem to exist in any of the other 3.x releases... `make install` does the trick there.

Reply:
The `make install` from 3.0 is now `make altinstall`, FWIW.